src/couch_replicator/src/couch_replicator_httpd_util.erl

```diff
@@ -158,7 +158,7 @@ docs_cb({meta, Meta}, #vacc{meta_sent = false, row_sent = false} = Acc) ->
     Parts =
         case couch_util:get_value(total, Meta) of
             undefined -> [];
-            Total -> [io_lib:format("\"total_rows\":~p", [adjust_total(Total)])]
+            Total -> [io_lib:format("\"total_rows\":~p", [Total])]
```

Comment:
Hi Jay,

is there a reason why we use io_lib:format instead couch_log: here?

Reply:
Yes, because it's not actually doing I/O here, but just [formatting the string](https://www.erlang.org/doc/man/io_lib.html#format-2). Are you thinking of `io:format/2` maybe?

Reply:
Ah, you are completly right! (_shakes fist at my brain... ;-)_)